Boards.h

```diff
@@ -278,6 +278,23 @@ writePort(port, value, bitmask):  Write an 8 bit port.
 #define PIN_TO_SERVO(p)         (p) // deprecated since v2.4
 
 
+// Arduino Nano 33 BLE
+#elif defined(ARDUINO_ARDUINO_NANO33BLE)
```

Comment:
FYI, this is not a typo.  The defined macro is `ARDUINO_ARDUINO_NANO33BLE`, for some reason.